src/templates.js

```diff
@@ -1,37 +1,25 @@
-var createAmpHeader = style => `
-  <!doctype html>
-  <html amp>
-    <head>
-      <meta charset="utf-8">
-      <link rel="canonical" href="default.html" >
-      <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
-      <style amp-custom>
-        ${style}
-      </style>
-      <style>body {opacity: 0}</style><noscript><style>body {opacity: 1}</style></noscript>
-      <script async src="https://cdn.ampproject.org/v0.js"></script>
-      <title>Index</title>
-    </head>
-`;
+const createAmpHeader = ({
+  headOverride, style, canonicalUrl, title, extraHeadHTML
+}) => `
+<!doctype html>
+<html amp>
+  <head>${typeof headOverride === 'string' ? headOverride : `
```

Comment:
is this a ***nested*** multi-line-string...? 
Talk about taking something to it's *logical conclusion*. :laughing: 
This is *probably* cleaner than an alternative approach ... :+1: